xml/art_migration.xml

```diff
@@ -834,22 +835,42 @@ ansible-playbook -i hosts/verb_hosts ardana-user-create.yml</screen>
      (after copying the ISO file into the user's home directory)
     </para>
 <screen>sudo mkdir -p /media/cloud
-sudo mount ~/HPE-HELION-OPENSTACK-*.iso /media/cloud</screen>
+echo $(ls ~/HPE-HELION-*.iso) /media/cloud iso9660 loop 0 0 \
+    | sudo tee -a /etc/fstab
+sudo mount -a</screen>
    </step>
    <step>
     <para>
-     Add the contents of the ISO as a Zypper repository and begin installation:
+     Add the contents of the ISO as a Zypper repository:
     </para>
 <screen>sudo zypper ar -G -f file:/media/cloud Cloud
 sudo zypper refresh</screen>
    </step>
    <step>
     <para>
-     Begin the Ardana.Pike installation:
+     Begin the installation:
     </para>
 <screen>sudo zypper -n in patterns-cloud-ardana
 sudo su - ardana
 ARDANA_INIT_AUTO=1 /usr/bin/ardana-init</screen>
+   </step>
+   <step>
+    <para>
+     Through this process, the directory
```

Comment:
This procedure creates the <filename>/srv/www/suse-12.3/x86_64/repos/PTF</filename> directory and adds it as a software repository. To enable a successful migration, you must add required Program Temporary Fix (PTF) RPM  packages to this repository manually. These packages are provide to you by be by &suse; via an active L3 support contract.